scripts/run_test_e2e_with_web.sh

```
@@ -0,0 +1,113 @@
+echo "Web app dir:     $CXR_WEBAPP_DIR"
+echo ""
+
+echo "Building and starting cloudxr-runtime..."
+"${COMPOSE[@]}" up --build -d cloudxr-runtime
+
+echo "Waiting for cloudxr-runtime to become healthy..."
+healthy=false
+for _ in $(seq 1 60); do
+    status="$("${COMPOSE[@]}" ps cloudxr-runtime --format '{{.Health}}' 2>/dev/null || true)"
+    case "$status" in
+        *healthy*)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 97.

(SC2221)